src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject.cs

```diff
@@ -39,7 +39,10 @@ internal sealed partial class CPSProject : AbstractProject
                 else if (!Path.IsPathRooted(binOutputPath))
                 {
                     // Make it a rooted path.
-                    binOutputPath = Path.Combine(Path.GetTempPath(), binOutputPath);
+                    var basePath = !string.IsNullOrEmpty(projectFilePath) && Path.IsPathRooted(projectFilePath) ?
```

Comment:
Oh - one more piece of feedback - we have our own `PathUtilities` class - should use be using that instead of `Path`?